azavea/echo-locator
# Replace requested Spanish translations

## Overview

This PR replaces relevant Spanish translations and fixes the resulting spacing issues.

## Testing Instructions

 * Start the app and navigate to **http://localhost:9966/**
 * Double check that some of the updated translations are appearing as they should in the app and that there are no spacing issues.


Resolves #467
